src/sql/azdata.d.ts

```diff
@@ -3739,7 +3739,7 @@ declare module 'azdata' {
 		 * Create a wizard page with the given title, for inclusion in a wizard
 		 * @param title The title of the page
 		 */
-		export function createWizardPage(title: string): WizardPage;
+		export function createWizardPage(title: string, pageName?: string): WizardPage;
```

Comment:
Don't modify azdata.d.ts when adding new stuff - add it to azdata.proposed.d.ts instead. Make sure to put the new function signature under the window namespace there. 

And for WizardPage just do this : 

```
export interface WizardPage {
/**
* An optional name for the page. If provided it will be used for telemetry
*/
pageName?: string;
}
```

and it'll be combined with the definition in azdata.d.ts.

Reply:
Thanks for the comment, Good to know that this file shouldn't be touched! have reverted the changes and made the changes in proposed file.